static/app/views/settings/account/accountNotificationFineTuning.tsx

```diff
@@ -26,7 +26,7 @@ import SettingsPageHeader from 'sentry/views/settings/components/settingsPageHea
 import TextBlock from 'sentry/views/settings/components/text/textBlock';
 
 const PanelBodyLineItem = styled(PanelBody)`
-  font-size: 1.4rem;
+  font-size: 1rem;
```

Comment:
Current font-size is 14px. This is not standard, we should use 16px (1rem in the new scale) instead.

<img width="718" alt="Screen Shot 2021-12-16 at 11 51 19 AM" src="https://user-images.githubusercontent.com/44172267/146445841-723958b7-521d-4e80-bb71-8561b268a26a.png">

Reply:
What did it look like before? Something weird is going on with that search input 😬

Reply:
Before:
<img width="528" alt="Screen Shot 2021-12-16 at 4 23 13 PM" src="https://user-images.githubusercontent.com/44172267/146468257-33e3d6cf-1d9b-476d-aa74-c9eac7c08f7f.png">

The search input isn't affected by this change. It's the panel item text ("javascript" & "sentry-mobile") that's currently at 14px and should be 16.